models/CouponModel.php

```diff
@@ -965,8 +978,12 @@ public function is_coupon_requirement_meet( $item_id, object $coupon, $order_typ
 		if ( self::REQUIREMENT_MINIMUM_QUANTITY === $coupon->purchase_requirement ) {
 			$min_quantity        = $coupon->purchase_requirement_value;
 			$is_meet_requirement = count( $item_ids ) >= $min_quantity;
+			if ( ! $is_meet_requirement ) {
```

Comment:
Here also, after filter result

Reply:
Setting the error message before or after the filter makes no difference!